Doc/library/stdtypes.rst

```diff
@@ -5221,16 +5222,11 @@ enables cleaner type hinting syntax compared to :data:`typing.Union`.
       TypeError: isinstance() argument 2 cannot be a parameterized generic
 
 The user-exposed type for the union object can be accessed from
-:data:`types.UnionType` and used for :func:`isinstance` checks.  An object cannot be
-instantiated from the type::
+:class:`typing.Union` and used for :func:`isinstance` checks::
 
-   >>> import types
-   >>> isinstance(int | str, types.UnionType)
+   >>> import typing
+   >>> isinstance(int | str, typing.Union)
    True
-   >>> types.UnionType()
-   Traceback (most recent call last):
-     File "<stdin>", line 1, in <module>
-   TypeError: cannot create 'types.UnionType' instances
```

Comment:
Maybe this should still be kept if you can't instantiate anymore

Reply:
Added it back